src/qt/veil.cpp

```diff
@@ -146,16 +146,12 @@ static void initTranslations(QTranslator &qtTranslatorBase, QTranslator &qtTrans
 /* qDebug() message handler --> debug.log */
 void DebugMessageHandler(QtMsgType type, const QMessageLogContext& context, const QString &msg)
 {
-#ifndef QT_MESSAGELOGCONTEXT
```

Comment:
Interesting.  Do you know where QT_MESSAGELOGCONTEXT is being defined?  This I added when I was debugging some other stuff, and I left it in because I had to specifically define QT_MESSAGELOGCONTEXT.  If it's getting defined somewhere else for MacOS (in our build), it should probably be removed from that instead.

Reply:
It is not defined anywhere else in the build system, nor used anywhere in the project.
Further, I'm having a bit of an issue with Github. let me fix this shortly.
Sorry!